src/javascript.coffee

```diff
@@ -515,7 +515,9 @@ define ['droplet-helper', 'droplet-model', 'droplet-parser', 'acorn'], (helper,
        node.type is 'segment'
       trailing trailing().replace(/;?\s*$/, '')
     else
-      trailing trailing().replace(/;?\s*$/, ';')
```

Comment:
The function that this is in (parens) already seems like it should/could be named better, considering adding semicolons doesn't really have to do with parens. Not sure what a good name is that covers both what happens in JS land and in coffee land.